website/assets/resources/install-fleetctl.sh

```diff
@@ -28,8 +28,8 @@ version_gt() {
 OS="$(uname -s)"
 
 case "${OS}" in
-    Linux*)     OS='linux';;
-    Darwin*)    OS='macos';;
```

Comment:
@noahtalerman The `OS` variable is used to extract the archive containing fleetctl and needs to be lowercase. I'll add a commit to this PR that adds an `OS_DISPLAY_NAME` variable that will be used in messages printed for the user.

Reply:
Nice! Thanks Eric